cypress/support/pages/admin/pageAdministration.js

```diff
@@ -269,5 +277,68 @@ class Management {
       .find('input')
       .click({ force: true })
   }
+
+  schoolNumberIsAdded() {
+    cy.get(Management.#schoolNumberForm)
+        .should('not.be.empty')
+  }
+
+
+  migrationButtonIsEnabled() {
+    cy.get(Management.#startMigrationButton)
+        .should('be.enabled')
+  }
+
+  clickStartMigration() {
+    cy.get(Management.#startMigrationButton)
+        .click()
+  }
+
+  migrationInformationIsVisible() {
+    cy.get(Management.#migrationInformationText)
+        .should('be.visible')
+  }
+
+  clickSupportLink() {
+    cy.contains("a", "Support")
+        .invoke("attr", "href")
+        .should('contain', 'mailto:nbc-support@netz-21.de')
+  }
+
+  clickInfoLink() {
+    cy.contains("[data-testid=\"text-description\"] > a", "hier")
+        .should("have.attr", "href", "https://blog.niedersachsen.cloud/umzug")
```

Comment:
Name of this method says that this is an action ("clickInfoLink"), but it is just checking, if the href attribute,.
Should be "checkInfoLink" or similar.